Design4/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Contact Page</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="icon">
+        <img class="icon" src="./contact-form-icon.svg" />
+      </div>
+      <div class="main">
+        <div class="headings">
+          <div class="heading1"><p>Interested in our business pricing?</p></div>
+          <div class="heading2">
+            <p>
+              Fill out the form to view details and we'll contact you as soon as
+              possible.
+            </p>
+          </div>
+        </div>
+      </div>
+        <form class="form1">
```

Comment:
it must not be from1 give it a name according to the form type like 'contact-us-form'